gui/src/components/ProcessList.svelte

```diff
@@ -121,7 +121,7 @@
               {/if}
             </td>
 
-            <td><h2>{name}</h2></td>
+            <td><button on:click={() => void router.push(`#/workspaces/${encodeURIComponent(workspaceId)}/processes/${encodeURIComponent(id)}`)}>{name}</button></td>
```

Comment:
can we use a Link component of some kind here? button seems like both the wrong semantics and appearance.